CONTRIBUTING.md

```diff
@@ -107,10 +107,32 @@ This section shows you how to send a pull request. The main staff are always mon
 
 This section shows you how to write commit messages in Clay. Follow these guidelines to help us maintain order and make it easier to locate your changes.
 
-#### Before You Make a Commit Message
+Each commit message consists of a header, a body and a footer. The header has a special format that includes a type, a scope and a subject:
 
--   Make sure that the problem or change you are making doesn't already exists.
--   Start the commit message with the word **Fixes**, and always follow it with a reference to the issue: - Fixes #0 - bla bla - Fixes #0, #1 - bla bla
+```
+<type>(<scope>): <subject>
+```
+
+The header is mandatory and the scope of the header is optional.
+
+> Clay follows the "[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)" specification.
+
+#### Type
+
+- **feat**: A new feature
+- **fix**: A bug fix
+- **docs**: Documentation only changes
+- **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+- **refactor**: A code change that neither fixes a bug nor adds a feature
+- **perf**: A code change that improves performance
+- **test**: Adding missing or correcting existing tests
+- **chore**: Changes to the build process or auxiliary tools and libraries such as documentation generation
+
+#### Scope
+
+The scope could be anything specifying place of the commit change. For example `feat(@clayui/dropdown)`, `feat(@clayui/css)`, `fix(next.clayui.com)`, `docs(Badge)`, `fix(useCache)`, etc...
+
+You can use `*` when the change affects more than a single scope.
```

Comment:
Is that `*` thing officially part of the spec? I don't see it mentioned. If it affects multiple places, shouldn't you just omit the scope, or pick something broader?

Reply:
Hum yeah, good question. Actually this is not in the specification but I see that this is more explicit than omit the scope, I think of it as a glob 🙂

Reply:
I think it's pretty visually noisy and redundant.

Reply:
And not spec compliant... 😁 

> A scope MUST consist of a noun describing a section of the codebase surrounded by parenthesis

`*` is not a noun

Reply:
Oh okay, I'll remove 😅